### PR TITLE
Reformat usage string

### PR DESCRIPTION
**Pull request by youssefragab99**
# Related issues

If applicable: N/A

Closes/Fixes/Addresses #<45>

If not applicable: write "N/A".

# Proposed changes

List out&mdash;with high level descriptions&mdash;what the commits
within this PR do:

- Reformat usage string to contain newline between argument and description
- Consistently formats all usage string descriptions to start with capital letter

### CODE REVIEW

File: codebasin.py [hunk 1]
```diff
@@ -66,23 +67,25 @@ def guess_project_name(config_path):
...
     )
     parser.add_argument(
         "-q",
         "--quiet",
         dest="quiet",
         action="count",
         default=0,
-        help="decrease verbosity level",
+        help="""
+    Decrease verbosity level""",
```
Comment (Pennycook) [inline codebasin.py:88]: This inserts a newline, but doesn't seem to do so consistently.

The "Increase" and "Decrease" lines have _extra_ indentation compared to other options:
```
  -r DIR, --rootdir DIR
                        Set working root directory (default .)
  -c FILE, --config FILE
                        Configuration file (default: <DIR>/config.yaml)
  -v, --verbose
                            Increase verbosity level
  -q, --quiet
                            Decrease verbosity level
```

**Review by Pennycook — CHANGES_REQUESTED** [review]
Thanks for taking a look at this. I think there are a few things that you need to address here.

### Sign-off Check

Thank you for reading the contributor guide.  I think you have misunderstood the intent of the sign-off check, though.  Please rewrite history to remove the commit with the certificate of origin.  Your remaining commit should look like this:
```
Add newlines to argparse help strings

Signed-off-by: Youssef Ragab <yragab99@gmail.com>
```

### Formatting

Please install and run the pre-commit hooks:
```
pre-commit install
pre-commit run
```

If you need to run on a specific file:
```
pre-commit run --files codebasin.py
```

There are some formatting changes in your patch that don't look right to me, and this should fix them.

### -h Option

The `-h` option still has the default string, with a lower-case "show":

```
optional arguments:
  -h, --help            show this help message and exit
```

File: codebasin.py [hunk 2]
```diff
@@ -49,8 +49,9 @@ def guess_project_name(config_path):
 if __name__ == "__main__":
     # Read command-line arguments
     parser = argparse.ArgumentParser(
-        description="Code Base Investigator v" + str(version),
-    )
+            description="Code Base Investigator v" + str(version),
```
Comment (laserkelvin) [inline codebasin.py:52]: `black` would modify this block

File: codebasin.py [hunk 3]
```diff
@@ -66,23 +67,25 @@ def guess_project_name(config_path):
...
+        help="Configuration file (default: <DIR>/config.yaml)",
     )
     parser.add_argument(
         "-v",
         "--verbose",
         dest="verbose",
         action="count",
         default=0,
-        help="increase verbosity level",
+        help="""
```
Comment (laserkelvin) [inline codebasin.py:78]: `black` should modify this block

File: codebasin.py [hunk 4]
```diff
@@ -92,22 +95,23 @@ def guess_project_name(config_path):
...
-        help="dump out annotated platform/parsing tree to <file.json>",
+        help="Dump out annotated platform/parsing tree to <file.json>",
     )
     parser.add_argument(
         "--batchmode",
         dest="batchmode",
         action="store_true",
         default=False,
-        help="Set batch mode (additional output for bulk operation.)",
+        help="""
```
Comment (laserkelvin) [inline codebasin.py:113]: `black` should modify this block

**Review by laserkelvin — CHANGES_REQUESTED** [review]
Echoing @Pennycook comments, particularly regarding the guidelines laid out in the contributor's guide. Additionally, the CI check fails because commits are unsigned, so we are unable to merge it until that is done.

**Comment by youssefragab99** [comment]
I see the issue, thank you both for the comments @laserkelvin @Pennycook, let me see if I can find a better way of doing this.

**Comment by Pennycook** [comment]
@youssefragab99 - Are you still working on this?  Our next release is planned for the end of March, and we were hoping to have this fix included there.

**Comment by Pennycook** [comment]
Closing as stale, and I'll open a new pull request to add the capital letters.  I'll defer the formatting until the next release. 
@youssefragab99, if you're still interested in working on the second part of the fix, please re-open the pull request.

**Pull request closed without merging**